example/tests/Provider/PactVerifyTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Provider;
+
+use GuzzleHttp\Client;
+use PhpPact\PactVerifier;
+use PhpPact\PactVerifierConfig;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Process\Process;
+
+class PactVerifyTest extends TestCase
+{
+    /** @var Process */
+    private $process;
+
+    protected function setUp()
+    {
+        $this->process = new Process('php -S localhost:8080 -t ../../src/Provider/public/');
```

Comment:
@nbrink91, this is to work around using a bootstrap?

Reply:
Yeah, let me clean that up a bit. I did that and didn't wrap it up. I need to add an assertion as well.